contents/dataVis.md

```diff
@@ -327,7 +324,7 @@ sco(s)
 
 Now, let's move on and do a plot with LaTeX strings and a custom theme.
 
-### LaTeXStrings
+### using LaTeXStrings
```

Comment:
I've forgotten our convention aren't we using `LaTeXStrings.jl` or `LaTeXStrings` for packages?

@rikhuijzer ?

Reply:
With jl, but Lazaro removed it here so that's okay. I guess it looks better in the menu without jl

Reply:
my bad. Yes with .jl, I will updated it.